server/scenario/web_api/scenario.py

```diff
@@ -1,13 +1,11 @@
 from flask import Blueprint
 
 import models
-from utils.decorator import admin_only
 
 web_api = Blueprint("web_api-scenario", __name__)
 
 
 @web_api.get("/templates")
-@admin_only
```

Comment:
Why not protect all endpoints even if they are "only" GET requests? Sensitive data might get leaked.